### PR TITLE
Task 4288 create ui search

### PR DESCRIPTION
**Pull request by vuanhmai2233**
Link redmine : https://edu-redmine.framgia.vn/issues/4288 .
Description : Create Ui tab search from remote .
photofacefun_com_1545893681

### CODE REVIEW

**Comment by vuanhmai2233** [comment]
ready

**Comment by tuanlvt-1117** [comment]
commented

**Comment by vuanhmai2233** [comment]
ready

File: app/src/main/java/com/framgia/music_44/screen/search_music/SearchFragment.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        initView(view);
+        setHasOptionsMenu(true);
+        return view;
+    }
+
+    private void initView(View view) {
+        Toolbar toolbar = view.findViewById(R.id.toolBar);
+        ((AppCompatActivity) getActivity()).setSupportActionBar(toolbar);
+        ((AppCompatActivity) getActivity()).getSupportActionBar().setTitle("Search");
+        toolbar.setTitleTextColor(Color.parseColor("#FFFFFF"));
```
Comment (tuanlvt-1117) [inline app/src/main/java/com/framgia/music_44/screen/search_music/SearchFragment.java]: hard code

File: app/src/main/res/layout/search_fragment.xml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        android:id="@+id/searchView"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        />
+    <android.support.v7.widget.RecyclerView
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        app:layout_constraintTop_toBottomOf="@id/searchView"
+        />
+</android.support.constraint.ConstraintLayout>
```
Comment (tuanlvt-1117) [inline app/src/main/res/layout/search_fragment.xml]: enter end line

File: app/src/main/res/menu/searchviewmenu.xml [hunk 3]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<menu xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+    <item
+        android:id="@+id/action_search"
+        android:icon="@drawable/ic_search_black_24dp"
+        android:orderInCategory="100"
+        android:title="Search ..."
```
Comment (tuanlvt-1117) [inline app/src/main/res/menu/searchviewmenu.xml]: hard code

**Comment by vuanhmai2233** [comment]
ready

**Review by tuanlvt-1117 — APPROVED** [review]

**Pull request merged**